<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="plyrTab">
     <attribute name="title">
      <string>Players</string>
     </attribute>
     <widget class="QTableView" name="plyrTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>771</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lastNameEdit">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>390</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="firstNameEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>390</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lastNameLbl">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>360</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="firstNameLbl">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>360</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>First Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="emailEdit">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>390</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="emailLbl">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>360</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="phoneNumEdit">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>390</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="phoneLbl">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>360</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Phone Number</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="addPlyrBtn">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>390</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Player</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="seasonTab">
     <attribute name="title">
      <string>Season</string>
     </attribute>
     <widget class="QPushButton" name="createSeasonBtn">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>390</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Season</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="seasonNameEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="seasonCB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="seasonPlyrView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>751</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="seasonLbl">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Season Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="meetTab">
     <attribute name="title">
      <string>Create Meet</string>
     </attribute>
     <widget class="QComboBox" name="meetSeasonCB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="meetPlyrView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>751</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="meetNameLbl">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Meet Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="meetNameEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="createGrpsBtn">
      <property name="geometry">
       <rect>
        <x>662</x>
        <y>390</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Groups</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addLatePlyrBtn">
      <property name="geometry">
       <rect>
        <x>552</x>
        <y>390</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Late Player</string>
      </property>
     </widget>
     <widget class="QPushButton" name="finalizeGrpsBtn">
      <property name="geometry">
       <rect>
        <x>442</x>
        <y>390</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Finalize Groups</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearGrpsBtn">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>390</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Groups</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Meet Groups</string>
     </attribute>
     <widget class="QComboBox" name="meetGrpsSeasonCB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="meetGrpsNameCB">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="meetGrpsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>751</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
